app/email/notification_email.go

```diff
@@ -44,3 +59,84 @@ func (es *Service) GetMessageForNotification(post *model.Post, translateFunc i18
 	}
 	return translateFunc("api.post.get_message_for_notification.files_sent", len(filenames), props)
 }
+
+/*
```

Comment:
For ease of reviewing, https://github.com/mattermost/mattermost-server/commit/31e6cdd0e6d334e35ca155e12a2d985b49af5f31 is the commit where these functions were added to `app/notification_email.go`

Reply:
Is this function the exact same as the one in `app/notification_email.go`? Can we reuse the one from `app/notification_email.go` instead of duplicating it?

Reply:
Made this the source instead, to avoid circular dependency.